phpunit.xml

```diff
@@ -19,19 +19,19 @@
             <file>./tests/e2e/Client.php</file>
             <directory>./tests/e2e/General</directory>
             <directory>./tests/e2e/Scopes</directory>
+            <directory>./tests/e2e/Services/Teams</directory>
+            <directory>./tests/e2e/Services/Realtime</directory>
+            <directory>./tests/e2e/Services/Webhooks</directory>
```

Comment:
iirc we moved the webhooks to the bottom like functions - because they take the longest

Reply:
Moved it to the end 👍